corehq/apps/accounting/models.py

```diff
@@ -1840,12 +1840,16 @@ def email_context(self):
             subscription__subscriber__domain=self.invoice.get_domain(),
         ))
         is_small_invoice = self.invoice.balance <= SMALL_INVOICE_THRESHOLD
+        payment_status = (_("Paid")
+                          if self.invoice.is_paid or total_balance == 0
```

Comment:
If there are other invoices that have a balance outstanding, `total_balance` would not be 0, but this particular invoice might be paid.